esprima.js

```diff
@@ -1274,27 +1281,13 @@
 
                     default:
                         if (isOctalDigit(ch)) {
```

Comment:
My vote is to follow the standard. That way, users know what to expect, and they can account for implementation bugs individually.